lib/App/index.js

```diff
@@ -2040,6 +2121,31 @@ class App {
       }
     }
 
+
+    /*
+    * If typescript is used, then add the types for node and add a .tsconfig.json file 
+    */
+    if (answers.typescript) {
+      await NpmCommands.install({ save: true }, {
+        id: '@types/node'
```

Comment:
We could also add `node-homey` right? Since it currently contains the sdk types.

Reply:
it's already in there ;-)

Reply:
Oh, makes sense, can't find it though, where is it added?